src/Config/VariationConfig.php

```diff
@@ -30,8 +33,13 @@ public static function fromArray(array $data): VariationConfig
         $id     = $data['id'] ?? 0;
         $name   = $data['name'] ?? '';
         $weight = $data['weight'] ?? 1;
+        $state  = ProjectState::PAUSED;
```

Comment:
Is this ProjectState? Or do they just share the same options?

Reply:
It's shared between them, I just didn't think of a good name. Let's go with RunState.